lib/context.ml

```diff
@@ -85,16 +95,17 @@ let refresh_state ctx =
       match get_local_file path with
       | Error e -> fmt_error "error while getting local file: %s\nfailed to get state from file %s" e path
       | Ok file ->
-        let state = State_j.state_of_string file in
+        let state = { ctx.state with state = State_j.state_of_string file } in
```

Comment:
later should move some of this fn into state.ml

Reply:
maybe leave that as a comment in code